.eslintrc.json

```diff
@@ -0,0 +1,162 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "es6": true,
+    "node": true,
+    "jest": true
+  },
+  "extends": ["plugin:react/recommended", "airbnb", "plugin:react-hooks/recommended"],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "plugins": [
+    "react",
+    "react-native",
+    "@typescript-eslint",
+    "react-hooks",
+    "prettier",
+    "json-format"
+  ],
+  "rules": {
+    "react-native/no-unused-styles": 2,
+    "react-native/split-platform-components": 2,
+    "react-native/no-inline-styles": 2,
+    "react-native/no-color-literals": 2,
+    "react-native/no-raw-text": 2,
+    "react-native/no-single-element-style-arrays": 2,
+
+    "react-native-a11y/has-accessibility-hint": 1,
+    "react-native-a11y/has-accessibility-props": 1,
+    "react-native-a11y/has-valid-accessibility-actions": 1,
+    "react-native-a11y/has-valid-accessibility-role": 1,
+    "react-native-a11y/has-valid-accessibility-state": 1,
+    "react-native-a11y/has-valid-accessibility-states": 1,
+    "react-native-a11y/has-valid-accessibility-component-type": 1,
+    "react-native-a11y/has-valid-accessibility-traits": 1,
+    "react-native-a11y/has-valid-accessibility-value": 1,
+    "react-native-a11y/no-nested-touchables": 1,
+    "react-native-a11y/has-valid-accessibility-descriptors": 1,
```

Comment:
where was this taken from? doesn't it have like a recommended set of rules by default?

Reply:
These rules came from [react-native-a11y](https://www.npmjs.com/package/eslint-plugin-react-native-a11y) @aguscha333 but I wasn't sure about default values for those rules so I preferred to specify them.

Reply:
cool